src/components/SocialShare/SocialShare.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { SocialShare, SocialShareProps } from './SocialShare';
+
+export default {
+  title: 'Components/Elements/SocialShare',
```

Comment:
Move this to live under Components/Inputs/Social Share so it's in the same place as button and drop down